crates/rust-analyzer/src/handlers.rs

```diff
@@ -871,6 +872,22 @@ pub(crate) fn handle_formatting(
     if !output.status.success() {
         match output.status.code() {
             Some(1) => {
+                // If `rustfmt` is a `rustup` proxy then it may exit 1 to indicate that the
```

Comment:
```suggestion
            Some(1) if !String::from_utf8(output.stderr).unwrap_or_default().contains("not installed") => {
                // If `rustfmt` is a `rustup` proxy then it may exit 1 to indicate that the
```

Reply:
Right now that'd prevent the silent swallowing of formatting errors